tests/foreman/ui/test_myaccount.py

```diff
@@ -16,96 +16,205 @@
 @Upstream: No
 """
 
+from fauxfactory import gen_string, gen_alpha
+from nailgun.entities import User
+from robottelo.constants import LANGUAGES
+from robottelo.datafactory import generate_strings_list
 from robottelo.decorators import stubbed, tier1
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators
+from robottelo.ui.locators import locators
+from robottelo.ui.session import Session
+
+
+def _valid_string_data(min_len=1, max_len=50):
+    """Generates valid string data for ui tests
+
+    :param min_len: min len for generated string
+    :param max_len: max len for generated string. Default is 50 because of
+    ui validation
+    :return: list of generated strings
+    """
+    return generate_strings_list(
+        min_length=min_len, max_length=max_len, exclude_types=['html']
```

Comment:
Why does html data is excluded? Is it intended behaviour or we have defect for that? In case of defect we should another skipping mechanism